.github/workflows/lambda-deploy.yml

```diff
@@ -0,0 +1,49 @@
+name: Deployment Workflow
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '20'
+
+      - name: Install npm dependencies
+        run: npm install
+
+      - name: Build NestJS application with Webpack
+        run: npm run webpack
+
+      - name: Print main.js Line Count
+        run: |
+          main_js_lines=$(wc -l < dist/main.js)
+          echo "Number of lines in main.js: $main_js_lines"
+
+      - name: Create deployment zip file
+        run: |
+          zip deployment.zip dist/main.js
```

Comment:
<img width="188" alt="image" src="https://github.com/shellygamlielly/tunity-server/assets/15690633/9a3a9884-1d90-454c-8cc6-ff39202157ae">
Thats probably correct since the config is `dist/main.handler`

Reply:
if you are correct it should be : " zip **-r** deployment.zip dist/main.js" this will create it in a "dist" directory. But if it worked for you we might not change it.